application-collabora-ui/src/main/resources/Collabora/Code/Main.xml

```diff
@@ -60,6 +60,22 @@
     &lt;/div&gt;
   &lt;/div&gt;
 #end
+#macro (unauthorizedUserModal)
+  &lt;div class="modal fade" id="unauthorizedUserPopup" tabindex="-1" role="dialog" aria-hidden="true"
```

Comment:
Popup is not the correct terminology, but  `unauthorizedUserModal` (you can update the method name if there are issues, e.g. `getUnauthorizedUserModal`)